app/src/main/java/com/example/timemanagementapp/ui/MainActivity.kt

```diff
@@ -127,7 +132,7 @@ class MainActivity : AppCompatActivity() {
              if(isNameDuplicated){
                  var indexOfDuplicatedName = syncedActivityNames.indexOf(newActivityName.toString())
                  var duplicateActivity = dailyActivityLog.activityLog.get(indexOfDuplicatedName)
-                 dailyActivityLog.activityLog.removeAt(indexOfDuplicatedName);
+                 dailyActivityLog.activityLog.removeAt(indexOfDuplicatedName)
```

Comment:
unnecessary

Reply:
Old habits can be hard to break... I still catch myself ending lines with ; periodically.  :)